blog/content/post/2020-04-03-trenchboot-nlnet-lz-validation.md

```diff
@@ -630,6 +668,95 @@ directory.
     be the same. It proves that LZ code utilizes SHA256 algorithm during
     measurements.
 
+##### check if LZ utilizes SHA1 algorithm when using TPM1.2 module
+
+1. If not already booted to `"NixOS - Secure Launch"`, reboot platform and boot
+to NixOS via `"NixOS - Secure Launch"` entry in GRUB menu.
+
+2. Update `landing-zone` package to have support for TPM1.2.
+
+    ```
+    $ cd ~/nixpkgs
+    $ git checkout tpm12_support
```

Comment:
Branch name will be changed when all changes will be merged in upstream. At this moment it is correct branch for TPM1.2 verification